samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<p>workspace = "/arcgis/home" </p><p> </p><p>@shivanip32 Should the workspace path be data/arcgis/home?</p><p>To make this sample running on try it live. I think we can upload a <strong>csv collection</strong> item and use it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
@shivanip32 Can you verify whether the `workspace` is correct and resolve this conversation if it is?